src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -304,7 +301,8 @@ public async Task InitializeAsync(CancellationToken cancellationToken = default)
 
                 // Initialize worker function invocation dispatcher only for valid functions after creating function descriptors
                 // Dispatcher not needed for non-proxy codeless function.
-                var filteredFunctionMetadata = functionMetadataList.Where(m => m.IsProxy() || !m.IsCodeless());
+                // Disptacher needed for non-dotnet codeless functions
+                var filteredFunctionMetadata = functionMetadataList.Where(m => m.IsProxy() || !Utility.IsCodelessDotNetLanguageFunction(m));
```

Comment:
@ankitkumarr - this allows loading "codeless" non-dotnet functions by the language worker. Let me know if anything else needs to be updated.